src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -762,7 +787,8 @@ private class FastqDemultiplexer(val sampleInfos: Seq[SampleInfo],
         )
       }
 
-    DemuxResult(sampleInfo=sampleInfo, numMismatches=numMismatches, records=demuxRecords)
+    val passQc = demuxRecords.forall(d => d.readInfo.forall(_.passQc))
+    DemuxResult(sampleInfo=sampleInfo, numMismatches=numMismatches, records=demuxRecords, passQc=passQc)
```

Comment:
Wouldn't it be a lot more efficient to do this up front?  What's the point of demuxing reads that we're just going to throw on the floor?  I would suggest doing this wherever `demultiplex()` is called from and then just skipping over the input fastq records.

Reply:
Leaving as is for now